content/blog/2021-12-08-apisix-integrate-skywalking-plugin/apisix-integrate-skywalking-plugin.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Apache APISIX Integrates with SkyWalking to Create a Full Fange of Log Processing"
+author: "Haochao Zhuang"
+authorURL: "https://github.com/dmsolr"
+authorImageURL: "https://avatars.githubusercontent.com/u/29735230?v=4"
+keywords: 
+- Apache APISIX
+- Apache SkyWalking
+- Log Process
+- Plugin
+description: This paper mainly introduces two Apache APISIX integrated SkyWalking log plug-ins to provide a more convenient operation and environment for log processing in Apache APISIX.
+tags: [Technology]
```

Comment:
I think we have different header from APISIX website

Reply:
sorry~I forgot this.